lib/domain/entity/receipt.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:equatable/equatable.dart';
+
+class Receipt extends Equatable {
+  final String? calories;
+  final String? carbos;
+  final String? description;
+  final int? difficulty;
+  final String? fats;
+  final String? headline;
+  final String? id;
+  final String? image;
+  final String? name;
+  final String? proteins;
+  final String? thumb;
+  final String? time;
+
+  const Receipt({
+    this.calories,
+    this.carbos,
+    this.description,
+    this.difficulty,
+    this.fats,
+    this.headline,
+    this.id,
+    this.image,
+    this.name,
+    this.proteins,
+    this.thumb,
+    this.time,
+  });
```

Comment:
Nit: Type should not be nullable since we're already know the response data and It's best to follow linter rules.  

Ref: https://dart.dev/tools/linter-rules/sort_constructors_first